docs/source/_includes/sawtooth-settings-list-pbft.inc

```diff
@@ -0,0 +1,20 @@
+* (PBFT only) Ensure that the on-chain setting
+  ``sawtooth.consensus.pbft.members`` lists the validator public keys of all
+  PBFT member nodes on the network.
+
+  a. Connect to the first validator node (the one that created the genesis
+     block).
+
+  #. Display the on-chain settings.
+
+     .. code-block:: console
+
+        $ sawtooth settings list
+
+  #. Verify that the output includes the public key for each node (not
+     including the first node that created the genesis block).
+
+     .. code-block:: console
+
+        sawtooth.consensus.pbft.members=03e27504580fa15...
```

Comment:
Put a bracket in here since it's a list: `...=[03e2...`

Reply:
Done.